src/main/app/controller/GetController.ts

```diff
@@ -68,12 +68,16 @@ export class GetController {
         ...content,
         ...renderableContents,
         sessionErrors: req.session.hasOwnProperty('errors') ? req.session.errors : [],
+        FileErrors: req.session.hasOwnProperty('fileErrors') ? req.session.fileErrors : [],
```

Comment:
Does this need to be FileErrors? Can it be camelCase?

Reply:
I should be able to change it.